docs/app/components/base-documentation-section/base-documentation-section.ts

```diff
@@ -1,34 +1,65 @@
-import { ISnippets } from '../../interfaces/ISnippets';
+import { ISnippets, SnippetCollection } from '../../interfaces/ISnippets';
+import { SiteThemeId } from '../../interfaces/SiteTheme';
 
 export abstract class BaseDocumentationSection {
 
     snippets: ISnippets;
+    theme: SiteThemeId = SiteThemeId.Keppel;
```

Comment:
The names are the strings which you can see in the dropdown in the top navigation bar. They are separate so that we can adjust those strings (which we have already had to do) without affecting the functionality of the application.

I'm not aware of any convention that says ID values have to be numbers.